src/model/database.py

```diff
@@ -27,3 +30,40 @@ def __validate__(schema_data):
     def __serialize_tables__(self):
         for table in self.tables:
             table.serialize()
+
+    @staticmethod
+    def get_schema(database_name):
+        path = os.path.join(Database.DATABASE_PATH, database_name)
+        if os.path.exists(path):
+            database = {}
+            database[Keys.DATABASE] = database_name
+            database[Keys.TABLES] = []
+            for table in os.listdir(path):
+                with open(os.path.join(path, table, "{}_schema.json".format(table))) as file:
+                    database[Keys.TABLES].append(json.load(file))
+            return database
+        raise WrongParameterError("Wrong database entered")
+
+    def get_table(self, table_name):
```

Comment:
Why you don’t save the table in dictionary instead of an array ?

Reply:
I'm more familiar working with arrays than with sets

Reply:
What about the performance ?
If you need to get it by order(1)

You should become familiar to deal with multiple data structures